src/core/server/saved_objects/mappings/types.ts

```diff
@@ -137,6 +137,7 @@ export interface SavedObjectsCoreFieldMapping {
   fields?: {
     [subfield: string]: {
       type: string;
+      ignore_above?: number;
```

Comment:
This seems to be only used in a test plugin and could probably be removed from the mapping instead (even if I did not try to launch the test suite with the mapping modified). But I guess it doesn't hurt to still add it.